docs/SETTINGS.md

```diff
@@ -68,6 +68,18 @@ Or if left empty
 This setting should be deprecated once the language server supports multiple workspaces,
 as this arises in VS code because a server instance is started per VS Code workspace.
 
+## `ignoreDirectoryNames` (`[]string`)
+
+This allows excluding directories from being checked by the walker by passing a static list of directory names.
```

Comment:
```suggestion
This allows excluding directories from being indexed upon initialization by passing a list of directory names.
```
Nitpick - Just a small suggestion to reflect that this is a user-facing doc and users may not know what a walker is.